src/dune_file.ml

```diff
@@ -1894,20 +1893,34 @@ module Copy_files = struct
 end
 
 module Documentation = struct
+  module Doc_Syntax = struct
+    type t = Reason | OCaml | All
+
+    let decode =
+      enum [ "reason", Reason
+           ; "ocaml", OCaml
+           ; "re", Reason
```

Comment:
I think that "reason" & "ocaml" are enough. These names are also used in formatting rules for examples.